src/components/ErrorMessages/ErrorMessages.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  hasError: boolean;
+  onCloseError: () => void;
+}
+
+export const ErrorMessages: React.FC<Props> = ({
+  hasError,
+  onCloseError,
+}) => {
+  useEffect(() => {
+    setTimeout(() => {
+      onCloseError();
+    }, 3000);
+  }, [hasError]);
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
+        { hidden: !hasError },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        aria-label="delete"
+        type="button"
+        className="delete"
+        onClick={onCloseError}
+      />
+
+      Unable to add a todo
+      <br />
+      Unable to delete a todo
+      <br />
+      Unable to update a todo
```

Comment:
You can remove these messages and add only message appropriate to the error. In the first task, you should implement appears error message in case loadTodos throw an Error